.github/workflows/deploy-dev.yaml

```
@@ -83,22 +83,29 @@ jobs:
       AUTH0_CLIENT_ID: ${{ secrets.AUTH0_CLIENT_ID }}
       AUTH0_CLIENT_SECRET: ${{ secrets.AUTH0_CLIENT_SECRET }}
       AUTH0_DOMAIN: ${{ vars.AUTH0_DOMAIN }}
       AUTH0_SECRET: ${{ secrets.AUTH0_SECRET }}
+      CLOUDFLARE_ACCOUNT_ID: ${{ vars.CLOUDFLARE_ACCOUNT_ID }}
+      CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+      DEPLOY_ENV: ${{ vars.DEPLOY_ENV }}
 
     permissions:
       contents: read
 
     steps:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 101: did not find expected key

(syntax-check)